build-tools-internal/src/main/groovy/elasticsearch.auto-backporting.gradle

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+
+import org.elasticsearch.gradle.internal.rewrite.RewritePlugin
+
+allprojects {
+    apply plugin: 'elasticsearch.rewrite'
+    rewrite {
+        rewriteVersion = "7.10.0"
```

Comment:
What is this version for?

Reply:
> I just had a run take 15 minutes, is that normal?
> 
> https://gradle-enterprise.elastic.co/s/gekq2dfwaxv5i
> 
> Is there any support for up-to-date checks with this processing?

We probably need to configure it to give the rewrite tasks more memory. That your server project rewrite task took 13 minutes is weired. Locally it took less than 3 minutes for me here. Likely because of the parallization of task execution.

Reply:
> What is this version for?



> Is it possible to skip files, e.g.
> 
> ```
> modules/lang-painless/src/test/java/org/elasticsearch/painless/RegexTests.java
> ```
> 
> The above file intentionally contains an invalid UTF-8 literal, which causes a compilation failure stacktrace to be printed to the console when running `rewrite`.

this seems like an issue with the rewrite java parser. I'm checking with the rewrite team on this

Reply:
> What is this version for?

this is the rewrite version we're using as base for this.